<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="titleLabel">
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>VEDA</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="idLabel">
     <property name="text">
      <string>아이디</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="idLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="pwLabel">
     <property name="text">
      <string>비밀번호</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="pwLineEdit"/>
   </item>
   <item>
    <widget class="QLabel" name="loginStatusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="text">
        <string>뒤로가기</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loginButton">
       <property name="text">
        <string>로그인</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
